ci/jenkins/jobs/macros.yaml

```diff
@@ -103,7 +103,7 @@
           #!/bin/bash
           set -ex
           DOCKER_REGISTRY="$(head -n1 ci/docker-registry)"
-          ./ci/jenkins/test-vmc.sh --cluster-name "$BUILD_TAG" --testcase e2e --coverage --codecov-token "${CODECOV_TOKEN}" --registry ${DOCKER_REGISTRY} --username "${CAPVC_USERNAME}" --password "${CAPVC_PASSWORD}"
+          ./ci/jenkins/test-vmc.sh --cluster-name "$BUILD_TAG" --testcase e2e --coverage --codecov-token "${CODECOV_TOKEN}" --registry ${DOCKER_REGISTRY} --username "${CAPVC_USERNAME}" --password "${CAPVC_PASSWORD}" --dockerUser "${{DOCKER_USERNAME}}" --dockerPassword "${{DOCKER_PASSWORD}}"
```

Comment:
Probably better to follow the same name pattern as others, e.g. '--docker-user' `--docker-password`

Reply:
Sure, updated.